tests/xtro-sharpie/xtro-report/Reporter.cs

```diff
@@ -0,0 +1,351 @@
+			if (full) {
+				log.WriteLine ($"<td align='center' bgcolor='green' width='{width}%'>Common</td>");
+				foreach (var platform in Platforms) {
+					log.WriteLine ($"<td align='center' bgcolor='green' width='{width}%'>{platform}</td>");
+					var files = Directory.GetFiles (InputDirectory, $"{platform}-*.ignore");
+					foreach (var file in files) {
+						var filename = Path.GetFileNameWithoutExtension (file);
+						var fx = filename.Substring (filename.IndexOf ('-') + 1);
+						if (!frameworks.Contains (fx))
+							frameworks.Add (fx); 
+					}
+				}
+			}
+			foreach (var platform in Platforms) {
+				log.WriteLine ($"<td align='center' bgcolor='red' width='{width}%'>{platform}</td>");
+				var files = Directory.GetFiles (InputDirectory, $"{platform}-*.unclassified");
+				foreach (var file in files) {
+					allfiles.Add (file);
+					var filename = Path.GetFileNameWithoutExtension (file);
+					var fx = filename.Substring (filename.IndexOf ('-') + 1);
+					if (!frameworks.Contains (fx))
+						frameworks.Add (fx); 
+				}
+			}
+			foreach (var platform in Platforms)
+				log.WriteLine ($"<td align='center' bgcolor='orange' width='{width}%'>{platform}</td>");
+			log.WriteLine ("</tr>");
+
+			var cols = (full ? 3 : 2) * Platforms.Length + (full ? 1 : 0);
+			log.WriteLine ("<tr>");
+			log.WriteLine ($"<td colspan='{cols + 1}' cellspanning='4'></td>");
+			log.WriteLine ("</tr>");
+
+			log.WriteLine ("</thead>");
+
+			var ignored = new int [Platforms.Length + 1];
+			var unclassified = new int [Platforms.Length];
+			var todo = new int [Platforms.Length];
+			int errors = 0;
+
+			frameworks.Sort ();
+			foreach (var fx in frameworks) {
+				if (Filter (fx))
+					continue;
+				log.WriteLine ("<tr>");
+				log.Write ("<td>");
+				if (!full && ProcessFramework (fx))
+					log.Write ($"<a href=\"{fx}.ignore.html\">{fx}</a>");
+				else
+					log.Write (fx);
+				log.WriteLine ("</td>");
+				if (full) {
+					string filename = $"common-{fx}.ignore";
+					var count = ProcessFile (filename);
+					log.Write ("<td align='center' ");
+					if (count < 1)
+						log.Write ("bgcolor='lightgreen'>-</td>");
+					else
+						log.Write ($"bgcolor='green'><a href=\"{filename}.html\">{count}</a>");
+					log.WriteLine ("</td>");
+					ignored [0] += count;
+					for (int i = 0; i < Platforms.Length; i++) {
+						filename = $"{Platforms [i]}-{fx}.ignore";
+						count = ProcessFile (filename);
+						log.Write ("<td align='center' ");
+						if (count < 1)
+							log.Write ("bgcolor='lightgreen'>-");
+						else
+							log.Write ($"bgcolor='green'><a href=\"{filename}.html\">{count}</a>");
+						log.WriteLine ("</td>");
+						ignored [i + 1] += count;
+					}
+				}
+				for (int i = 0; i < Platforms.Length; i++) {
+					string filename = $"{InputDirectory}/{Platforms [i]}-{fx}.unclassified";
+					var count = ProcessFile (filename);
+					log.Write ("<td align='center'");
+					if (count < 1)
+						log.Write (" bgcolor='salmon'>-");
```

Comment:
🐟

Reply:
glad it did not went unnoticed ;-)
and now we can say there's something fishy about that PR :-)

Reply:
:iseewhatyoudidthere: